src/flags.ts

```diff
@@ -116,30 +116,29 @@ export class LDFlagManager implements IFlagManager {
 	constructor(ctx: vscode.ExtensionContext, settings: IConfiguration) {
 		this.settings = Object.assign({}, settings);
 		let config = this.config(settings);
-		if (settings.sdkKey) {
-			this.updateProcessor = StreamProcessor(settings.sdkKey, config, Requestor(settings.sdkKey, config));
-			this.start();
-		} else {
-			vscode.window.showWarningMessage(
-				'[LaunchDarkly] sdkKey is not set. LaunchDarkly language support is unavailable.',
-			);
+		if (!settings.sdkKey) {
+			console.warn('LaunchDarkly sdkKey is not set. Language support is unavailable.');
```

Comment:
Log a warning instead of showing a message in the case where the extension is not yet configured (https://github.com/launchdarkly/ld-vscode/issues/18)